frontend/src/components/common/Header.tsx

```diff
@@ -8,51 +9,163 @@ import {
   Flex,
   Icon,
   IconButton,
+  Image,
   Link,
   Stack,
   Text,
   useDisclosure,
 } from "@chakra-ui/react";
-import React from "react";
-import { Link as ReactLink } from "react-router-dom";
+import React, { useContext, useEffect } from "react";
+import { Link as ReactLink, useHistory } from "react-router-dom";
 
-import { LOGIN_PAGE } from "../../constants/Routes";
-import { MenuIcon } from "./icons";
+import authAPIClient from "../../APIClients/AuthAPIClient";
+import {
+  HOME_PAGE,
+  LANDING_PAGE,
+  LOGIN_PAGE,
+  SCHEDULING_PAGE,
+} from "../../constants/Routes";
+import AuthContext from "../../contexts/AuthContext";
+import { CloseIcon, MenuIcon } from "./icons";
 
 const Header = (): JSX.Element => {
+  const history = useHistory();
   const { isOpen, onOpen, onClose } = useDisclosure();
+  const { authenticatedUser, setAuthenticatedUser } = useContext(AuthContext);
+
+  const onLogOutClick = async () => {
+    const success = await authAPIClient.logout(authenticatedUser?.id);
+    if (success) {
+      setAuthenticatedUser(null);
+      history.push(LANDING_PAGE);
+    }
+  };
 
   return (
-    <Container pt="0.5rem">
-      <Flex>
+    <Container pt="0.5rem" maxWidth={{ base: "default", md: "70%" }}>
```

Comment:
nit: let's do 1.5 rem so it looks a bit closer to figma designs
![image](https://user-images.githubusercontent.com/19617248/141868545-74bcfc54-2185-4a12-a2b3-4a1b348784e9.png)

```suggestion
    <Container pt="1.5rem" maxWidth={{ base: "default", md: "70%" }}>
```